app/browser/tabs.js

```diff
@@ -302,7 +302,8 @@ const fixDisplayURL = (navigationEntry, controller) => {
     navigationEntry.virtualURL = getSourceAboutUrl(navigationEntry.virtualURL)
   }
 
-  if (isIntermediateAboutPage(navigationEntry.virtualURL)) {
+  if (isIntermediateAboutPage(navigationEntry.virtualURL) &&
+    navigationEntry.virtualURL.indexOf('about:safebrowsing#') === -1) {
```

Comment:
doh- missed your previous comments 😛  (needed to refresh). Let me implement a better check 😄